### PR TITLE
Restructrue ledger and rosetta docs

### PR DESCRIPTION
**Pull request by domwoe**
Signed-off-by: Dominic Woerner <dominic.woerner@dfinity.org>

First PR that restructures the Ledger and Rosetta docs and adds a few introductory words and clarifications.
This PR will be followed by another PR that adds an "Interact with the ledger" page that provides practical advice for developers to interact with the ICP ledger and links to sample code.

@MarioDfinity Please have a look if this is an improvement from your perspective

### CODE REVIEW

**Comment by MarioDfinity** [comment]
Nice, thank you @domwoe ! I have to work on the working group today but I'll have a look tomorrow for sure. Jean should also review this.

File: docs/developer-docs/functionality/ledger/deploy-new-token.md [hunk 1]
```diff
@@ -101,11 +101,11 @@ This tutorial will guide you step-by-step to deploy your own token to the IC and
 
 Your new token is deployed and ready to be used.
 
-## Connect rosetta-api
+## Connect Rosetta
```
Comment (jean-dfinity) [inline docs/developer-docs/functionality/ledger/deploy-new-token.md:104]: Is it an official name change? it brings the question of the name of _our_ Rosetta vs the official naming:

> Rosetta is an open-source specification and set of tools that makes integrating with blockchains simpler, faster, and more reliable.

> The Rosetta APIs make a set of explicit design choices to enable simpler, faster, and more reliable blockchain integrations.

_https://www.rosetta-api.org/docs/welcome.html_

File: docs/developer-docs/functionality/ledger/ledger-local-setup.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # Ledger Local Setup
 
+If you are working in a local development environment, i.e with a local replica instead of the public Internet Computer, you can't access the ICP ledger. In order to test your application that integrates with the ICP ledger locally, you need to deploy a local ledger canister. Howver, this local ledger canister won't have the history and balances of the live ICP ledger.
```
Comment (jean-dfinity) [inline docs/developer-docs/functionality/ledger/ledger-local-setup.md:3]: typo (Howver)

File: docs/developer-docs/functionality/ledger/deploy-new-token.md (hunk 1 above)
Comment (domwoe) [reply docs/developer-docs/functionality/ledger/deploy-new-token.md:104]: Ah, I think I got it. Here you mean the specific rosetta-api application that was built for  the ledger canister? In this case it makes sense to revert the change.

File: docs/developer-docs/functionality/rosetta/index.md [hunk 3]
```diff
@@ -1 +1,224 @@
...
+
+Not yet. Before launch, when we push to the `dfinity/rosetta-api:latest` image, it’s usually a major update that we’ll announce in our communication channels beforehand.
+
+We’ll soon implement nightly builds of the image, and CI will ensure it works before pushing. Other than `latest`, those images will also be tagged with the build date, so for more reproducibility, it’s possible to use the image of a specific date tag rather than `latest`. We’ll announce when nightly builds become available.
+
+#### How do I connect to the main net instead of the test net?
+
+Start `dfinity/rosetta-api` with `--help`, you can see some additional CLI arguments that can be passed. Among those there are `--canister-id` and `--ic-url` which can be used to configure the ledger destination. At the moment, they default to the test net.
+
+**Note**: The main net is not live yet; it will be live some time before the publicly announced date, and we’ll push the updated image to point to the main net to ensure you can perform testing on the main net beforehand.
```
Comment (jean-dfinity) [inline docs/developer-docs/functionality/rosetta/index.md:122]: Probably not required anymore?

Comment (domwoe) [reply docs/developer-docs/functionality/rosetta/index.md]: Good catch. Removed it.

File: docs/developer-docs/functionality/ledger/ledger-local-setup.md (hunk 2 above)
Comment (domwoe) [reply docs/developer-docs/functionality/ledger/ledger-local-setup.md]: Fixed

**Review by MarioDfinity — COMMENTED** [review]
@domwoe this goes in the right direction imho. I assume there is some work to be done, e.g. filling the "Interact with the ICP ledger" section and also change `ledger/index.md`, right? Is it going to be done as part of this PR?

**Comment by MarioDfinity** [comment]
@domwoe it looks good. It's an improvement already and we should merge it when you are ready. I like the idea of making docs easier to read from newcomers.

**Pull request merged**